src/platform/api/pythonApi.ts

```diff
@@ -76,13 +76,17 @@ export function pythonEnvToJupyterEnv(env: ResolvedEnvironment): PythonEnvironme
             envType = EnvironmentType.VirtualEnv;
         }
     }
+    if (!env.executable.uri) {
```

Comment:
One quick question. Is this a logging case? Like should we log an info message? Not sure if this is expected to happen often or more rare.

Reply:
We might want to discuss with Python team on this. We might need to document/log what partial python environment means in each scenario. Let's tackle this in Nov.